PanoptesNetClient/PanoptesNetClient/Auth.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Net;
+using System.Net.Http;
+
+namespace PanoptesNetClient
+{
+    public class Auth
+    {
+        //  string BearerToken;
```

Comment:
Are you going to use there? If not, they should probably be deleted

Reply:
These will likely be used once the `Auth` class has full functionality.